{"id": "121687", "title": "Bug 121687: Exception thrown when exporting data in a table inside a grid cell.", "description": "When trying to export data in a table that is inside a grid cell, an exception is thrown. The issue occurs while executing the 'ReportEngineService.getMetaData()' method.", "OB": "An exception is thrown when trying to export data in a table inside a grid cell.", "EB": "When exporting data in two tables, the data should be successfully exported.", "SR": "To reproduce the issue: \n1. Insert a grid. \n2. Attempt to export data in two tables."}